python-avd/pyavd/_eos_designs/structured_config/base/daemon_terminattr.py

```diff
@@ -28,10 +28,6 @@ def daemon_terminattr(self: AvdStructuredConfigBaseProtocol) -> None:
 
         The schema will enforce that we only use either new or old models.
         """
-        if not self.inputs.cv_settings:
-            self._legacy_daemon_terminattr()
-            return
```

Comment:
why do you remove the early return? You should keep

```
        if not (cv_settings := self.inputs.cv_settings):
            return
```

and you can then remove line 35

Reply:
Added lines back..